### PR TITLE
new docs

### PR DESCRIPTION
**Pull request by helenlly**
## What changes were proposed in this pull request?

add docs structure and original md files

### CODE REVIEW

**Comment by yiheng** [comment]
We should put all image in a resource/images folder

**Comment by yiheng** [comment]
what about the font issue?

**Comment by yiheng** [comment]
Please add a readme.md in the doc folder to explain how to generate a website from this doc folder

File: docs/docs/userguide/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+  The parameters used in the above command are:
+
+  * -f: The folder where your put the CIFAR-10 data set. Note in this example, this is just a local file folder on the Spark driver; as the CIFAR-10 data is somewhat small (about 120MB), we will directly send it from the driver to executors in the example.
+
+  * -b: The mini-batch size. The mini-batch size is expected to be a multiple of *total cores* used in the job. In this example, the mini-batch size is suggested to be set to *total cores * 4*
+
+## Next Steps
+* To learn the details of Python support in BigDL, you can check out the [Python Support Page](https://github.com/intel-analytics/BigDL/wiki/Python-Support)
```
Comment (shane-huang) [inline docs/docs/userguide/getting-started.md]: There're still hyper links to wiki pages. Is there any cross refrence utlity for refering to pages in the same site?

File: docs/docs/userguide/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+* To learn how to create practical neural networks using BigDL in a couple of minutes, you can check out the [Tutorials Page](https://github.com/intel-analytics/BigDL/wiki/Tutorials)
```
Comment (shane-huang) [inline docs/docs/userguide/getting-started.md]: Hyper links to wiki pages needs to be modified. Wiki pages will be removed later.

File: docs/docs/userguide/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+* you can check out the [Document Page](https://github.com/intel-analytics/BigDL/wiki/Documents) for more details (including Tutorials, Examples, Programming Guide, etc.)
```
Comment (shane-huang) [inline docs/docs/userguide/getting-started.md]: Hyper links to wiki pages needs to be modified. Wiki pages will be removed later.

File: docs/docs/userguide/models.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+   * [VGG](https://github.com/intel-analytics/BigDL/tree/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/models/vgg): it demonstrates how to use BigDL to train and evaluate a [VGG-like](http://torch.ch/blog/2015/07/30/cifar.html) network on CIFAR-10 data.
+   * [ResNet](https://github.com/intel-analytics/BigDL/tree/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/models/resnet): it demonstrates how to use BigDL to train and evaluate the [ResNet](https://arxiv.org/abs/1512.03385) architecture on CIFAR-10 data.
+   * [RNN](https://github.com/intel-analytics/BigDL/tree/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/models/rnn): it demonstrates how to use BigDL to build and train a simple recurrent neural network [(RNN) for language model](http://www.fit.vutbr.cz/research/groups/speech/publi/2010/mikolov_interspeech2010_IS100722.pdf).
+   * [Auto-encoder](https://github.com/intel-analytics/BigDL/tree/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/models/autoencoder): it demonstrates how to use BigDL to build and train a basic fully-connected autoencoder using MNIST data.
+
+## **Examples**
```
Comment (shane-huang) [inline docs/docs/userguide/models.md]: Shall we change this to "Scala Examples"? We have "Python Examples" below.

File: docs/docs/userguide/models.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+# **Models**
```
Comment (shane-huang) [inline docs/docs/userguide/models.md]: Why examples are put into Models Page? A little bit confusing.

File: docs/docs/userguide/faq.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+# **FAQ**
```
Comment (shane-huang) [inline docs/docs/userguide/faq.md]: The contents of this page look more like "Trouble Shooting and Tips" than "FAQ"

Comment (jason-dai) [inline docs/docs/userguide/faq.md]: Let's change it back to Known Issues

**Comment by zhichao-li** [comment]
Maybe we should add an image folder to hold pictures of example
i.e: ELU
image

**Comment by yiheng** [comment]
why there're two user guide?

**Comment by yiheng** [comment]
can we put the build website command into the readme.md?

**Comment by yiheng** [comment]
can we keep the folder name lower case?

**Pull request merged**